pcsx2/Counters.h

```diff
@@ -85,17 +85,17 @@ struct SyncCounter
 //------------------------------------------------------------------
 // SPEED HACKS!!! (1 is normal) (They have inverse affects, only set 1 at a time)
 //------------------------------------------------------------------
-#define HBLANK_COUNTER_SPEED	1 //Set to '3' to double the speed of games like KHII
+#define HBLANK_COUNTER_SPEED	2 //Set to '3' to double the speed of games like KHII
```

Comment:
Definitely not.

Reply:
forgot to flip that back whoops was just messing around